CHANGELOG.unreleased.md

```diff
@@ -103,3 +103,62 @@ Examples:
   (a?.b[c]).c();
   let value = /** @type {string} */ (this.members[0]).functionCall();
   ```
+
+- JavaScript: Respect newlines between parameters ([#5260] by [@evilebottnawi])
+
+  <!-- prettier-ignore -->
+  ```js
+  // Input
+  function foo(
+    one,
+  
+    two,
+    three,
+    four,
+  
+  
+    five,
+    six,
+    seven,
+    eight,
+    nine,
+    ten,
+  
+    eleven
+  
+  ) {}
+    
+  // Output (Prettier stable)
+  function foo(
+    one,
+    two,
+    three,
+    four,
+    five,
+    six,
+    seven,
+    eight,
+    nine,
+    ten,
+    eleven
+  ) {}
+    
+  // Output (Prettier master)
+  function foo(
+    one,
+  
+    two,
+    three,
+    four,
+  
+  
```

Comment:
maybe leaving maximum one empty line?

Reply:
@lipis yep, we can, i just think it is better explain changes

Reply:
fixes, also add snapshot to ensure